tempodb/encoding/common/config.go

```diff
@@ -30,6 +30,9 @@ type BlockConfig struct {
 
 	// parquet fields
 	RowGroupSizeBytes int `yaml:"parquet_row_group_size_bytes"`
+
+	// vParquet3 fields
+	DedicatedColumns backend.DedicatedColumns `yaml:"dedicated_columns"`
```

Comment:
I see a some potential issues with this approach:
* If we use the `parquet3_` prefix it will be confusing as soon as the default encoding is `vParquet4` or higher but still  supporting the same dedicated column configuration
* Currently the assumption is that we will remove support for `v2` in the near future. Therefore, a `parquet_` prefix will become pretty meaningless as soon as every encoding is based on parquet.

Reply:
> Currently the assumption is that we will remove support for v2 in the near future.

We have committed to supporting `v2` for the foreseeable future for those users who want to run Tempo as a super high performance key/value store for traces.

Reply:
OK, I added a `parquet_` prefix

@mapno We need to change this in the [vParquet3 docs PR](https://github.com/grafana/tempo/pull/2664) too

Reply:
Done, thanks for the headsup